spec.html

```diff
@@ -27159,34 +27201,44 @@ <h1>Example Cyclic Module Record Graphs</h1>
             <table>
               <thead>
                 <tr>
-                  <th>Module</th>
-                  <th>[[DFSIndex]]</th>
-                  <th>[[DFSAncestorIndex]]</th>
-                  <th>[[Status]]</th>
-                  <th>[[AsyncEvaluation]]</th>
-                  <th>[[AsyncParentModules]]</th>
-                  <th>[[PendingAsyncDependencies]]</th>
-                  <th>[[EvaluationError]]</th>
+                  <th>Fields / Modules</th>
+                  <th>_A_</th>
+                  <th>_C_</th>
                 </tr>
               </thead>
               <tr>
-                <th>_A_</th>
+                <th>[[DFSIndex]]</th>
                 <td>0</td>
-                <td>0</td>
-                <td>~evaluated~</td>
-                <td>*true*</td>
-                <td>« »</td>
-                <td>1 (_B_)</td>
-                <td>~empty~</td>
+                <td>3</td>
               </tr>
               <tr>
-                <th>_C_</th>
-                <td>3</td>
+                <th>[[DFSAncestorIndex]]</th>
+                <td>0</td>
                 <td>0</td>
+              </tr>
+              <tr>
+                <th>[[Status]]</th>
                 <td>~evaluated~</td>
+                <td>~evaluated~</td>
+              </tr>
+              <tr>
+                <th>[[AsyncEvaluation]]</th>
+                <td>*true*</td>
                 <td>*true*</td>
+              </tr>
+              <tr>
+                <th>[[AsyncParentModules]]</th>
+                <td>« »</td>
                 <td>« _A_ »</td>
+              </tr>
+              <tr>
+                <th>[[PendingAsyncDependencies]]</th>
+                <td>1 (_B_)</td>
                 <th>0</th>
```

Comment:
Is this a `<th>` on purpose? This is what it looks like on the spec. 
![2405_lxgf8](https://github.com/tc39/ecma262/assets/782056/ad54ccac-83ef-4990-a2a7-a571956c252c)

Reply:
seems like a typo (weird that the `]]` isn't sticking together on the same line, btw)

Reply:
Almost certainly not.

Reply:
@ljharb outside my focus, but fwiw that’s what it looks like in firefox. Here’s some examples of table 45 in 16.2.1.5.4

Safari:
![2406_0qawe](https://github.com/tc39/ecma262/assets/782056/75acd5e3-3ccb-4bf3-8d16-f6beb6e70be0)

Chrome:
![2406_0il40](https://github.com/tc39/ecma262/assets/782056/2b496c24-20ca-4a45-9406-1c12289f5246)

Firefox:
![2406_ddlnc](https://github.com/tc39/ecma262/assets/782056/eb8409c4-67e2-45c9-ac39-9d9a858b4c13)

Reply:
Yeah that `th` is for sure a typo.